src/Canvas/Canvas.js

```diff
@@ -0,0 +1,213 @@
+import * as React from "react";
+
+import { Box } from "@mui/material";
+import Grid from "./Grid";
+import PropTypes from "prop-types";
+import ResizeHandle from "./ResizeHandle";
+import SelectionRectangle from "./SelectionRectangle";
+import useSelectionRectangle from "./useSelectionRectangle";
+
+/**
+ * A canvas component that can be used for as part of a drawing application. It includes features such as resize, grid, and selection rectangle.
+ */
+const Canvas = React.forwardRef(
+  (
+    {
+      backgroundColor = "white",
+      backgroundImage = "",
+      border = true,
```

Comment:
Same here. showBorder might be better. You would instinctively expect border to align with the css border prop '1px solid red' etc.